src/App.tsx

```diff
@@ -1,203 +1,60 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { Todo } from './types/Todo';
+import { FilterOptions } from './types/FilterOptions';
+import { Errors } from './types/Errors';
+import { TodoApp } from './components/TodoApp';
+import { TodoList } from './components/TodoList';
+import { TodosFilter } from './components/TodosFilter';
+import { TodoError } from './components/TodoError';
+import { getTodos } from './api/todos';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterOption, setFilterOption] = useState<FilterOptions>(
+    FilterOptions.all,
+  );
+  const [errorText, setErrorText] = useState<Errors | null>(null);
+
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => setTodos(todosFromServer))
```

Comment:
```suggestion
      .then(setTodos)
```
we can simplify that